src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,5 @@
+.logo {
+  position: absolute;
+  margin-top: 10px;
+  margin-left: 50px;
+}
```

Comment:
Don't set margins for BEM block. Add additional class for logo to make it the element of other block

Reply:
Use header__logo class and set all margins for it